deployment/refinery-modules/refinery/manifests/neo4j-ontologies.pp

```diff
@@ -0,0 +1,89 @@
+class refinery::neo4j-ontologies {
+  package { 'unzip':
+    name        => 'unzip',
+    ensure      => latest,
+    require     => Class['apt::update'],
+  }
+  exec { "stop neo4j service":
+    command     => "sudo service neo4j-service stop",
+    user        => $app_user,
+    group       => $app_group,
+    path        => ['/usr/bin/'],
+    require     => Service["neo4j-service"],
+  }
+  exec { "remove old db":
+    command     => "sudo rm -rf graph.db",
+    cwd         => "/var/lib/neo4j/data/",
+    user        => $app_user,
+    group       => $app_group,
+    path        => ['/usr/bin/'],
+    require     => Exec["stop neo4j service"],
+  }
+  exec { "fetch pre-generated db":
+    command     => "sudo wget -q http://data.cloud.refinery-platform.org.s3.amazonaws.com/data/stem-cell-commons/neo4j/2015/graph.db.zip && yes | sudo unzip graph.db.zip",
```

Comment:
Also, we use a [Puppet module](https://forge.puppet.com/puppet/archive) for these operations.